lib/screens/signup/signup_with_phone/signup_with_phone.dart

```diff
@@ -21,6 +22,14 @@ class _SignupWithPhoneState extends State<SignupWithPhone> {
     });
   }
 
+  bool validateIndianPhone(String value) {
```

Comment:
isValidIndianPhone is a much better name

Reply:
updated